workflow/snakefile

```diff
@@ -35,6 +35,10 @@ def get_otoole_data(otoole_config: str, var: str) -> list[str]:
         "TotalDiscountedStorageCost",
         "ModelPeriodEmissions",
         "StorageLowerLimit",
+        "CapitalInvestmentStorage",
+        "DiscountedCapitalInvestmentStorage",
+        "DiscountedCostByStorage",
+        "DiscountedSalvageValueStorage"
```

Comment:
You probably saw, but this may have conflicts with the changes we developed in PR #219 (the summaries). I believe the ones in summaries are correct (which may overlap with these)